apps/web/pages/thank-you.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <NuxtLazyHydrate when-visible>
+    <ConfirmationPageContent v-if="data" :order="data" />
+    <SoftLogin v-if="error" :error="error" @submit="loadOrder" />
```

Comment:
when is the softlogin component shown? only when error?
is this the intended behaviour ?
"Orders that are from a guest user promt them to give their name. If name isn't available, ask for zip-code"